datavec/datavec-python/src/main/java/org/datavec/python/PythonContextManager.java

```diff
@@ -118,7 +115,7 @@ private static void expandContext(String contextName) {
 
     }
 
-    public static void setContext(String contextName) {
+    public static void setContext(String contextName) throws Exception{
```

Comment:
Please add javadoc for this. Explain what a "context" is. Even if you have to do an at see or put the main docs in one method or class header then reference those.